.github/workflows/build.yml

```diff
@@ -439,11 +438,11 @@ jobs:
           cd ${CWD}
 
       - name: Release
-        if: ${{ steps.release-charts.outputs.tag != '' }}
```

Comment:
One thing to note here is that this step will transparently just _skip_ in cases where webCatalogOnly is an unexpected value (for whatever reason). I'm not sure how I feel about it.

With that said, I do very much prefer what you're changing this to as a criteria because it is way more descriptive.

This comment may be a no-op, as I'm not sure where I land on this. Feel free to disregard if it doesn't do anything to your opinion on what's being changed here (which is totally fine).

Reply:
Let's agree the webCatalogOnly should never take another value than "True" or "False". Otherwise this is a bug that needs to be a) found, b) fixed in its time.